docs/source/How_to_install_liminal_in_airflow_on_kubernetes.md

```diff
@@ -0,0 +1,123 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required bgit y applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Install Liminal in Airflow
+* [Workflow](#workflow)
+* [Prerequisites](#prerequisites)
+   * [Supported Distributions](#supported-distributions)
+* [Get Started](#Get-Started)
+   * [Package dependency](#package-dependency)
+   * [Mount the EFS](#mount-the-EFS)
+   * [Liminal deployment](#liminal-deployment)
+* [References and other resources](#references-and-other-resources)
+
+## Workflow
+To deploy and run liminal on airflow, we need to complete two tasks:
+1. Ensure that the Liminal python package is installed inside each one of Airflow's pods (schedulers and Web Server)
+2. Ensure that the user's Liminal code (i.e. Yamls) are present in each of the pods' DAGs folder (typically located in /opt/airflow/dags)
+
+The first task is accomplished by iterating over Airflows' k8s pods and running a pip install on each of them.
+
+The second task is accomplished by using a folder on a shared filesystem (EFS) to host the airflow DAGs (and Liminal Yamls). \
+This folder is mounted into Airflow pods' DAGs folder - thus enables each of the pods to pick the files up and run the Liminal DAGs.
+
+The process of liminal is emphasized on the following diagram:
+
+![](assets/liminal_deployment_diagram.png)
+
+## Prerequisites
+Before you begin, ensure you have met the following requirements: 
+* You have the kubectl command line [(kubectl CLI)][homebrew-kubectl] installed
+* You have the current [context][cluster-access-kubeconfig] in kubernetes' kubeconfig file
+* You have [Airflow on Kubernetes with AWS EFS][airflowInstallation]
+
+### Supported Distributions
+
+|Distribution | Versions |
+|-|-|
+|[Airflow][airflowImage] | apache/airflow:1.10.14-python3.8 |
+|[Airflow Helm Chart][airflowChart] | [7.16.0][airflow-helm-chart-7.16.0] |
+
+#### You may need to add the Repo
+
+```sh
+helm repo add airflow-stable https://airflow-helm.github.io/charts
+helm repo update
+```
+
+## Get Started
+#### Package dependency
+There are multiple ways to install liminal in Airflow and it depends on what will be useful in your environment.\
+1. Use kubectl to execute pip install on the pods.
+2. Add `apache-liminal` as a requirement package to be installed once Airflow starts.
```

Comment:
3. Build your own Airflow images based on the official docker image from https://github.com/apache/airflow/blob/0d3ee669243a5c5648fa7d707cd3547905772142/Dockerfile, 
with the command 
docker build --build-arg ADDITIONAL_PYTHON_DEPS='apache-liminal'